regression/contracts/loop-history-variables-04/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--apply-loop-contracts
+^EXIT=(10|6)$
+^SIGNAL=0$
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.1\] .* assertion x == z: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+This test checks that __CPROVER_loop_entry is supported.
```

Comment:
How exactly are we checking that `__CPROVER_loop_entry` is supported? It doesn't appear in the the C program ...